scalafmt-core/shared/src/main/scala/org/scalafmt/util/TreeOps.scala

```diff
@@ -137,6 +155,8 @@ object TreeOps {
         case t: Defn.Macro => addDefn[KwDef](t.mods, t)
         case t: Decl.Def => addDefn[KwDef](t.mods, t)
         case t: Defn.Enum => addDefn[KwEnum](t.mods, t)
+        case t: Defn.ExtensionGroup =>
+          addDefnTokens(Nil, t, "extension", t => soft.KwExtension.unapply(t))
```

Comment:
i may have asked. why do we need a lambda here? can't we just use KwExtension.unapply directly?

Reply:
The problem is that `extension` is a soft keyword so a tokenizer treats it as an Ident and `ident` will not match `extension`

Reply:
ok, i am not sure i understood. what i was asking was whether this code would work equally:
```
  addDefnTokens(Nil, t, "extension", soft.KwExtension.unapply)
```
that is, instead of `t => soft.KwExtension.unapply(t)`, can we use the simpler `soft.KwExtension.unapply`. and if we can't, can we at least use a different variable (because the case also uses `t`, so it might be a bit confusing)?

Reply:
Ach, sorry! I totally misread it - or in reality I somehow never really read it :O  Changed!